lib/matplotlib/axes/_base.py

```diff
@@ -3902,18 +3902,18 @@ def _make_twin_axes(self, *kl, **kwargs):
 
     def twinx(self):
         """
-        Create a twin Axes sharing the xaxis
+        Create a twin Axes instance sharing the x-axis with the parent 
```

Comment:
is it that unclear? maybe "parent" isn't the exact terminology, but after all `twinx()` is a method called on an existing `Axes`. I am going to revert this part of the docstring edit.